charts/nextcloud/values.yaml

```diff
@@ -330,6 +330,15 @@ persistence:
   ##
   # storageClass: "-"
 
+  ## Use a additional pvc for the data directory rather than a subpath of the default pvc
+  ## Useful to store data on a different storageClass (e.g. on slower disks)
+  nextcloudData:
+    enabled: false
+    subPath:
+    # storageClass: "-"
+    accessMode: ReadWriteOnce
```

Comment:
Can we add the `existingClaim` example here?

```yaml
# existingClaim: "-"
```

Reply:
Hi @tvories  added the same existingClaim here and moved the whole section to the bottom of `persistence`.
Is it okay this way or should i also include the ` "-"` ?

Reply:
No, this is great!  Thanks!